k4GaudiPandora/CMakeLists.txt

```
@@ -21,15 +21,15 @@ set(_plugin_sources ${CMAKE_CURRENT_LIST_DIR}/
 					# src/components/*.cpp
 					src/CalorimeterHitType.cc
 					# src/DDBFieldPlugin.cc
-					# src/DDCaloDigi.cc
+					src/DDCaloDigi.cc
 					# src/DDCaloHitCreator.cc
 					# src/DDExternalClusteringAlgorithm.cc
 					# src/DDGeometryCreator.cc
 					# src/DDMCParticleCreator.cc
 					# src/DDPandoraPFANewProcessor.cc
 					# src/DDPfoCreator.cc
-					# src/DDScintillatorPpdDigi.cc
-					src/DDSimpleMuonDigi.cc
+					src/DDScintillatorPpdDigi.cc
+					# src/DDSimpleMuonDigi.cc
```

```suggestion
					src/DDSimpleMuonDigi.cc
```
Here we don't want to just disable `DDSimpleMuonDigi.cc` from being compiled. Can you also remove here all the files that don't exist yet and are commented out? Then they can be added as needed.